OnlineSubsystemBasic/Source/Private/OnlineIdentityInterfaceBasic.cpp

```diff
@@ -285,3 +285,10 @@ void FOnlineIdentityBasic::Tick(float DeltaTime)
 {
 
 }
+
+void FOnlineIdentityBasic::GetLinkedAccountAuthToken(int32 LocalUserNum, const FOnGetLinkedAccountAuthTokenCompleteDelegate& Delegate) const
+{
+	FExternalAuthToken AuthToken;
+	AuthToken.TokenString = GetAuthToken(LocalUserNum);
+	Delegate.ExecuteIfBound(LocalUserNum, AuthToken.HasTokenString(), AuthToken);
```

Comment:
Same question around if we should use `Subsystem->ExecuteNextTick` for this

Reply:
This behavior duplicates the Steam OSS behavior with wrapping GetAuthToken() in 5.1.  Though generally I prefer execute next tick to prevent recursion, I figured I would go with at least one of the engine standards.